cameo/flux_analysis/analysis.py

```diff
@@ -676,6 +676,13 @@ def plot(self, grid=None, width=None, height=None, title=None, axis_font_size=No
         if grid is None:
             plotter.display(plot)
 
+    @staticmethod
+    def _axis_label(variable_id, nice_variable_id, unit):
+        if "BIOMASS" in variable_id:
```

Comment:
obviously won't always work but it's an improvement - make case insensitive to cover a couple of more situations?